tests/Temporalio.Tests/Worker/WorkflowWorkerTests.cs

```diff
@@ -7210,6 +7210,86 @@ public async Task ExecuteWorkflowAsync_PayloadMissing_StillWorks()
         await replayer.ReplayWorkflowAsync(history);
     }
 
+    [Workflow]
+    public class LocalActivityMissingWorkflow
+    {
+        [WorkflowRun]
+        public async Task RunAsync()
+        {
+            await Workflow.ExecuteLocalActivityAsync("invalid_activity", Array.Empty<object>(), new() { ScheduleToCloseTimeout = TimeSpan.FromHours(1) });
+        }
+
+        [Activity]
+        public static string? DoSomething() => null;
+    }
+
+    [Fact]
+    public async Task ExecuteWorkflowAsync_LocalActivityMissing()
+    {
+        var err = await ExecuteWorkerAsync<LocalActivityMissingWorkflow, WorkflowHandle>(
+            async worker =>
+            {
+                var handle = await Client.StartWorkflowAsync(
+                    (LocalActivityMissingWorkflow wf) => wf.RunAsync(),
+                    new($"workflow-{Guid.NewGuid()}", worker.Options.TaskQueue!));
+                await AssertTaskFailureContainsEventuallyAsync(handle, "Activity invalid_activity is not registered on this worker, available activities: DoSomething");
+                return handle;
+            },
+            new TemporalWorkerOptions().AddAllActivities<LocalActivityMissingWorkflow>(null));
```

Comment:
```suggestion
        await ExecuteWorkerAsync<LocalActivityMissingWorkflow>(
            async worker =>
            {
                var handle = await Client.StartWorkflowAsync(
                    (LocalActivityMissingWorkflow wf) => wf.RunAsync(),
                    new($"workflow-{Guid.NewGuid()}", worker.Options.TaskQueue!));
                await AssertTaskFailureContainsEventuallyAsync(handle, "Activity invalid_activity is not registered on this worker, available activities: DoSomething");
            },
            new TemporalWorkerOptions().AddAllActivities<LocalActivityMissingWorkflow>(null));
```

Pedantic, but no need for `err` or `return` of the handle or any of that here (or in other tests below)